console

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env php
+<?php
+define('SKIP_HM_DISPATCH', true);
+
+
+// Load the application dependencies and bootstrap the application
+require_once __DIR__ . '/index.php';
```

Comment:
Load the dependencies here and don't include index.php. Treat console as a separate entry point in the application. index.php will be for web requests and this one for cli.